bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -1,8 +1,11 @@
 package edu.gemini.catalog.api
 
-import edu.gemini.spModel.core.{Angle, Coordinates}
+import edu.gemini.spModel.core.Coordinates
+import edu.gemini.spModel.core.Target.SiderealTarget
 
-case class CatalogQuery(base: Coordinates, coneRadius: Angle, catalog: CatalogName = sdss)
+case class CatalogQuery(base: Coordinates, radiusConstraint: RadiusConstraint, magnitudeConstraints: MagnitudeConstraints, catalog: CatalogName = ucac4) {
+  def filter: SiderealTarget => Boolean = (t) => radiusConstraint.targetsFilter(base)(t) && magnitudeConstraints.filter(t)
+}
```

Comment:
There may be some confusion (in my head at least).  There are two distinct concepts here.  There's the question of whether other (non-gemini) catalog servers must be supported vs whether only a single catalog being hosted by a server need be supported. My understanding is that no other servers need be supported (and so no UI choice) but that we will still need a few of the catalogs.  AGS of course has its own defaults and doesn't prompt the user but a manual search still might provide catalog options.  I guess we should seek clarification here.